src/components/NavBar/NavBar.module.css

```diff
@@ -1,3 +1,9 @@
+.navbar {
+  position: sticky;
+  z-index: 900;
+  background-color: var(--color-white);
```

Comment:
Hmm the navbar is still missing a box shadow (Bug 4). You might just need to change the background color to something else.